ios/RNInstabug/InstabugSessionReplayBridge.h

```diff
@@ -20,6 +21,12 @@
 
 - (void)getSessionReplayLink:(RCTPromiseResolveBlock)resolve :(RCTPromiseRejectBlock)reject;
 
+- (void)setSyncCallback;
+
+- (void)evaluateSync:(BOOL)result;
+
+@property (nonatomic, copy) SessionEvaluationCompletion sessionEvaluationCompletion;
```

Comment:
`atomic` property adds performance cost due to the locking mechanism, do we really need to add that without explicit requirement that we need to make this property thread safe or do we have a general requirement for thread safety ?

Reply:
@kholood-ea The cost shouldn't be significant. I think it's needed in this context since we don't have control over the thread the sync callback will be called in and don't have the guarantee that it will be the same one as the thread for `evaluateSync`.